src/components/Square.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+const Square = ({
+  i,
+  squares,
+  setSquares,
+  onClick,
```

Comment:
Is this used?

Reply:
Correct, I don't need onClick in Square component (nor have it passed down in Board or App components).